packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/GroupsSection/GroupsSection.scss

```diff
@@ -6,32 +6,6 @@
   gap: 24px;
   width: 100%;
 
-  .empty-groups-container {
```

Comment:
whie remove 🤔?

Reply:
according to [this ](https://github.com/hicommonwealth/commonwealth/pull/8622/files#diff-923f2c6c210470036b809eb402e17666c58d77a953b2e5529ac382f47c5ca967L50), it shouldn't exist when the allowlist is available. I will double check with @kurtisassad

Reply:
Yeah basically in the figma the empty groups banner was removed.

Reply:
thank you Kurtis!